cmd/nfs-subdir-external-provisioner/provisioner.go

```diff
@@ -155,14 +159,12 @@ func (p *nfsProvisioner) Delete(ctx context.Context, volume *v1.PersistentVolume
 	}
 
 	// Determine if the "onDelete" parameter exists.
-	// If it exists and has a delete value, delete the directory.
-	// If it exists and has a retain value, safe the directory.
+	// If it exists and has a deleted value, delete the directory.
+	// If it exists and has a retail value, safe the directory.
```

Comment:
n_: retain

Reply:
oh, corrected and made those values to be explicit in the comments. @kmova ptal. thanks